src/Components/JobsContainer/JobsContainer.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from 'react';
+import {
+    IJobsContainerProps,
+    IJobsContainerStyleProps,
+    IJobsContainerStyles
+} from './JobsContainer.types';
+import { getStyles } from './JobsContainer.styles';
+import { classNamesFunction, CommandBar, styled } from '@fluentui/react';
+import { useExtendedTheme } from '../../Models/Hooks/useExtendedTheme';
+import JobsList from './Internal/JobsList/JobsList';
+import JobsDialog from './Internal/JobsDialog/JobsDialog';
+import useAdapter from '../../Models/Hooks/useAdapter';
+import { AdapterMethodParamsForJobs, IAdtApiJob } from '../../Models/Constants';
+
+const getClassNames = classNamesFunction<
+    IJobsContainerStyleProps,
+    IJobsContainerStyles
+>();
+
+const JobsContainer: React.FC<IJobsContainerProps> = ({ adapter, styles }) => {
+    // contexts
+
+    // state
+    const [isJobsDialogOpen, setIsJobsDialogOpen] = useState(false);
+    const [listOfJobs, setListOfJobs] = useState<Array<IAdtApiJob>>([]);
+
+    const getAllJobs = useAdapter({
+        adapterMethod: () => adapter.getAllJobs(),
+        refetchDependencies: [adapter],
+        isAdapterCalledOnMount: false
+    });
+    // hooks
+    const deleteJob = () => {
+        useAdapter({
+            adapterMethod: (params: AdapterMethodParamsForJobs) =>
+                adapter.deleteJob(params),
+            refetchDependencies: [adapter],
+            isAdapterCalledOnMount: false
+        });
+    };
+
+    const cancelJob = () => {
+        useAdapter({
+            adapterMethod: (params: AdapterMethodParamsForJobs) =>
+                adapter.cancelJob(params),
+            refetchDependencies: [adapter],
+            isAdapterCalledOnMount: false
+        });
+    };
+
+    // callbacks
+
+    // side effects
+    useEffect(() => {
+        getAllJobs.callAdapter();
```

Comment:
Since in the stories you are not passing any adapter. so not sure if those methods are all working in runtime. with getAllJobs.callAdapter() on mount I think you should be getting an error since there is no adapter passed to your stories